final.js

```diff
@@ -1,52 +1,62 @@
-function Book(title, author, id) {
-  this.title = title;
-  this.author = author;
-  this.id = id;
+/* eslint-disable max-classes-per-file */
+class Book {
+  constructor(title, author, id) {
+    this.title = title;
+    this.author = author;
+    this.id = id;
+  }
 }
 
 let myBooks = [];
 const container = document.querySelector('.book-container');
+const form = document.querySelector('#form');
```

Comment:
- [ ] please try to use JavaScript classes instead of global objects and arrays, you can use objects and arrays but it has to be inside a class.